_posts/databases/2000-01-01-scalingo-elasticsearch-addon.md

```diff
@@ -80,10 +80,33 @@ SCALINGO_ELASTICSEARCH_URL=http://example-app-3030:MpIxXstskccB3Ab2iwKH@example-
 
 If you need to access your database from other places than your app please follow the [Access your database]({% post_url databases/2015-06-24-access-database %}) guide.
 
+### Force TLS connections
+
+Our Elasticsearch image support TLS to encrypt all of its network traffic
+between the client and the server.
+
+Elasticsearch cannot listen to connections with and without TLS. If you want to
+encrypt communications with your Elasticsearch databases, you need to force all
+connections to use TLS. Forcing TLS connections is as simple as heading to the
+database dashboard and clicking on the toggle button:
+
+{% assign img_url = "https://cdn.scalingo.com/documentation/screenshot_database_mongo_force_tls.png" %}
+{% include mdl_img.html %}
+
+Note that you must have configured your application to use TLS when connecting
+to the database.
+
+```shell
+> curl -X GET https://<URL>
```

Comment:
This looks weird. Why not curl -X GET <URL> and specifing that the URL should begin with https://?